Production/govuk_ios/SupportingFiles/AppDelegate.swift

```diff
@@ -8,7 +8,6 @@ class AppDelegate: UIResponder,
                      didFinishLaunchingWithOptions
                      launchOptions: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool {
         analyticsService.configure()
-        analyticsService.setAcceptedAnalytics(accepted: true)
```

Comment:
Highlighting this removal as seems this function call was initially added to always enable analytics for dev/test purposes as there was previously no other way of enabling? 
Removing this now that we can enable and disable analytics on app startup using Analytics Consent screen.